modules/acs-requirements.adoc

```diff
@@ -1,32 +1,16 @@
 // Module included in the following assemblies:
 //
-// * installing/prerequisites.adoc
+// * installing/acs-requirements.adoc
 :_module-type: CONCEPT
```

Comment:
```suggestion
:_content-type: CONCEPT
```

Reply:
Fixed it! Tnx! I have noticed that we have a mix of `_module-type` and `_content-type` in our repo.